### PR TITLE
LPS-160852 SF

### PR DESCRIPTION
**Pull request by seltong**

### CODE REVIEW

**Comment by seltong** [comment]
ci:test:relevant

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:relevant

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:relevant

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:relevant

**Comment by seltong** [comment]
ci:test:relevant

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:relevant

File: modules/apps/object/object-service/src/main/resources/resource-actions/default.xml [hunk 1]
```diff
@@ -30,9 +30,12 @@
 		<weight>1</weight>
 		<permissions>
 			<supports>
+				<action-key>ADD_DISCUSSION</action-key>
 				<action-key>DELETE</action-key>
+				<action-key>DELETE_DISCUSSION</action-key>
 				<action-key>PERMISSIONS</action-key>
 				<action-key>UPDATE</action-key>
+				<action-key>UPDATE_DISCUSSION</action-key>
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-service/src/main/resources/resource-actions/default.xml:38]: Hi @seltong,

Set DELETE_DISCUSSION and UPDATE_DISCUSSION in guest-unsupported

thanks!

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/object_entry/discussion.jspf [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%
+ObjectLayoutBox commentsObjectLayoutBox = objectEntryDisplayContext.getObjectLayoutBoxByType(ObjectLayoutBoxConstants.TYPE_COMMENTS);
+%>
+
+<c:if test="<%= !objectEntryDisplayContext.isReadOnly() && objectDefinition.isEnableComments() && (commentsObjectLayoutBox != null) %>">
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/object_entry/discussion.jspf:21]: Create a method:

isDuscussionEnable

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Comment by seltong** [comment]
ci:test:sf

**Review by MarinhoFeliphe — APPROVED** [review]
LGTM!! :tada:

**Comment by MarinhoFeliphe** [comment]
ci:forward

**Comment by seltong** [comment]
ci:forward

**Comment by seltong** [comment]
ci:forward

**Pull request closed without merging**